tests/e2e/manifest/send_traffic.sh

```
@@ -121,25 +123,32 @@ echo "${CYAN}--- Starting Traffic Test ---${NC}"
 failure_count=0
 
 for i in $(seq 1 $MAX_RETRIES); do
+    if [ $i -lt 2 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)